### PR TITLE
[v1.3] Logic:Ui:Docs: Theme feature

### PR DESCRIPTION
**Pull request by yewshengkai**
Add new theme feature.
theme 1 (no theme)
theme 2 (blue theme)
theme 3 (dark theme)

- [ ] More themes to be added
- [x] Test case to be added

### CODE REVIEW

**Comment by karrui** [comment]
Fix the issues raised by [Codacy](https://www.codacy.com/app/karrui/main/pullRequest?prid=1016683), easy and should prevent regressions!

**Comment by yewshengkai** [comment]
yep not ready. Still updating it.

File: src/main/java/seedu/address/logic/commands/ThemeCommand.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ * Selects a person identified using it's last displayed index from the address book.
+ */
+public class ThemeCommand extends Command {
+
+    public static final String COMMAND_WORD = "theme";
+    public static final String COMMAND_ALIAS = "t";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Switches to selected theme\n"
+            + "1. No theme\n"
```
Comment (karrui) [inline src/main/java/seedu/address/logic/commands/ThemeCommand.java]: Maybe call it `1. Default theme` instead? But this one is up to you

Comment (yewshengkai) [reply src/main/java/seedu/address/logic/commands/ThemeCommand.java]: oki. I update it next v1.4. If not gonna change a lot with given time left.

**Pull request merged**